extensions/wikia/ImageReview/ImageReviewDatabaseHelper.class.php

```diff
@@ -23,24 +25,30 @@ public function selectImagesForList( $sOrder,
 		$oDB = $this->getDatawareDB( DB_SLAVE );
 
 		$oResults = $oDB->query('
-			SELECT pages.page_title_lower, image_review.wiki_id, image_review.page_id, image_review.state, image_review.flags, image_review.priority, image_review.last_edited
-			FROM (
-				SELECT image_review.wiki_id, image_review.page_id, image_review.state, image_review.flags, image_review.priority, image_review.last_edited
-				FROM `image_review`
-				WHERE state = ' . $iState . ' AND top_200 = 0
```

Comment:
This query ended up getting moved into the `ImageListGetter` class and you'll notice the join was actually removed altogether. We weren't using any of the results from the `pages` table, so I just removed it.